tests/testing/test_exampleJob.py

```diff
@@ -30,9 +30,7 @@ def tearDownClass(cls):
 
     def test_delete_existing_job(self):
         self.job = self.project.create_job("ExampleJob", "job_test_delete_existing_job")
-        with warnings.catch_warnings(record=True) as w:
-            self.job.run(run_again=True)
-            self.assertEqual(len(w), 1)
```

Comment:
It is strange that the warning message disappeared.

Reply:
I assume  dac53480d2f9b200e0c4b77e60df97a17f7831b3 is the culprit, will have a look.

Reply:
It's a deeper bug in the deprecator, I'll close this and bump the pyiron_base version again after I fix it there.